### PR TITLE
Docs: Add customer defined calibration

### PR DESCRIPTION
**Pull request by feng-intel**
Docs: Add customer defined calibration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,14 +87,21 @@ from neural_compressor.torch.quantization import (
...
 qconfig = FP8Config(fp8_config="E4M3")
 model = prepare(model, qconfig)
+
 # customer defined calibration
-calib_func(model)
+model(torch.randn(1, 3, 224, 224).to("hpu"))
+
 model = convert(model)
+
+output = model(torch.randn(1, 3, 224, 224).to("hpu"))
```
Comment (xin3he) [inline README.md:103]: output = model(torch.randn(1, 3, 224, 224).to("hpu")).cpu()

File: README.md [hunk 2]
```diff
@@ -87,14 +87,21 @@ from neural_compressor.torch.quantization import (
...
 )
+
+import torch
 import torchvision.models as models
 
 model = models.resnet18()
 qconfig = FP8Config(fp8_config="E4M3")
 model = prepare(model, qconfig)
+
 # customer defined calibration
```
Comment (xin3he) [inline README.md:98]:
```suggestion
# customer defined calibration, below is a dummy calibration
```

File: README.md [hunk 3]
```diff
@@ -87,14 +87,21 @@ from neural_compressor.torch.quantization import (
...
-calib_func(model)
+model(torch.randn(1, 3, 224, 224).to("hpu"))
+
 model = convert(model)
+
+output = model(torch.randn(1, 3, 224, 224).to("hpu"))
+print(output.shape)
```
Comment (xin3he) [inline README.md:104]: What's the intention of this print?

File: README.md [hunk 4]
```diff
@@ -87,14 +87,21 @@ from neural_compressor.torch.quantization import (
     prepare,
     convert,
 )
+
```
Comment (xin3he) [inline README.md:90]: Could you please help update the docker image:
`docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none --cap-add=sys_nice --net=host --ipc=host vault.habana.ai/gaudi-docker/1.18.0/ubuntu22.04/habanalabs/pytorch-installer-2.4.0:latest`

**Review by xin3he — APPROVED** [review]

**Review by thuang6 — APPROVED** [review]

**Pull request merged**